plugins/locator/stdloc/stdloc.cpp

```diff
@@ -309,20 +309,23 @@ bool StdLoc::init(const Config::Config &config) {
 	                                      0.100, 0.200, 0.400};
 	defaultProf.enableConfidenceEllipsoid = true;
 	defaultProf.confLevel = 0.9;
-	defaultProf.gridSearch.autoLatLon = true;
 	defaultProf.gridSearch.originLat = 0.;
 	defaultProf.gridSearch.originLon = 0.;
-	defaultProf.gridSearch.originDepth = 5.;
-	defaultProf.gridSearch.xExtent = 20.;
-	defaultProf.gridSearch.yExtent = 20.;
-	defaultProf.gridSearch.zExtent = 5.;
-	defaultProf.gridSearch.cellXExtent = 2.5;
-	defaultProf.gridSearch.cellYExtent = 2.5;
-	defaultProf.gridSearch.cellZExtent = 5.0;
+	defaultProf.gridSearch.originDepth = 20.;
+	defaultProf.gridSearch.autoOriginLon   = true;
```

Comment:
Are these boolean parameters really needed? The behaviour can be directly inferred from the configuration of originLat, ...Lon, and ...Depth.

Reply:
o.k.